themes/courses.labster.com/lms/templates/emails/enroll_email_ccx_coach_enrolledmessage.txt

```diff
@@ -3,7 +3,7 @@ from django.conf import settings
 from django.utils.translation import ugettext as _
 %>
 
-${_("Dear {full_name}").format(full_name=full_name)}
+${_("Hi there,")}
```

Comment:
Will there be space after the comma in resulting sentence?

Reply:
@TamoshaytisV  no